code/modules/food/recipes_microwave_ds.dm

```diff
@@ -1,3 +1,86 @@
+/datum/recipe/microwave/bacon
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/rawbacon
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/bacon
+
+/datum/recipe/microwave/baconcheeseburger
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/cheeseburger,
+		/obj/item/weapon/reagent_containers/food/snacks/bacon
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/baconcheeseburger
+
+/datum/recipe/microwave/baconcheeseburger
```

Comment:
2 recipes shouldn't have the same type

Reply:
Oh ok I was going for a multiple recipe approach but forgot to even test both versions in game. When it likely fails, could of maybe looked at the combinable burger or sandwich code or something. I'll get on it, or will just choose which recipe to go with for the moment.